crates/bevy_winit/src/lib.rs

```diff
@@ -783,7 +845,7 @@ pub fn winit_runner(mut app: App) {
 
     trace!("starting winit event loop");
     if return_from_run {
-        run_return(&mut event_loop, event_handler);
+        pump_events(&mut event_loop, event_handler);
```

Comment:
I'm nooot so sure about it, see https://github.com/rust-windowing/winit/pull/2767 for detailed winit API changes.

Reply:
mmmm, no, I doubt you want to use `pump_events()` in bevy.

You should be able to either use the regular `run` API now (which supports returning now - except on iOS/Web) or else use `run_on_demand()` which behave a lot more like `run_return` did.

`pump_events` is for cases where your game engine implements it's own external loop which would also need to have some mechanism for throttling that loop.